frontend/src/components/applyForm/ApplyFormMessage.tsx

```diff
@@ -0,0 +1,55 @@
+import { useTranslations } from "next-intl";
+import { Alert } from "@trussworks/react-uswds";
+
+import { FormValidationWarning } from "./types";
+
+export const ApplyFormMessage = ({
```

Comment:
[nice to have] tests for this?

Reply:
All of the states are tested in the applyForm tests.